## Radna sveska - Python
### Python kao kalkulator


Dodaj nove ćelije i izračunaj vrednosti sledećih brojevnih izraza:


a) 2 + 5




b) $2 ^ {10}$ (<i>napomena: može se otkucati kao 2 ** 10 </i>)






c) 1000 - 98 * 12





d) 52 / 6  (<i>deljenje</i>)


<h3>Promenljive - imena međurezultata</h3>
Ponekad će nam biti potrebno da zapamtimo pojedine međurezultate. Pogledaj sledeći primer:

<p>Ukupan broj učenika u svim okruzima Srbije je 249656, a broj učenica je 123281. Odrediti da li u Srbiji ima više učenica ili učenika. (Podaci preuzeti sa <a href="http://opendata.mpn.gov.rs/index.php?ucenici_studenti=srednje"> linka</a> - otvoreni podaci MPNTR.</p> 

<p>Uradimo zajedno ovaj primer.</p>

<p>Označimo podatke koje smo dobili na sledeći način:</p>

In [5]:
ukupan_broj_ucenika = 249656
broj_ucenica = 123281

Broj učenika ćemo izračunati kada od ukupnog broja učenika oduzmemo broj učenica.

In [6]:
broj_ucenika = ukupan_broj_ucenika - broj_ucenica

Ako želiš da vidiš koliki je ukupan broj učenika u Srbiji, to možeš učiniti tako što otkucaš sledeće:

In [10]:
broj_ucenika

126375

Sada možemo da zaključimo da učenica ima 123281, a učenika 126375, što znači da je učenika više.

In [ ]:
Šta bi bio rezultat izvršavanja sledećeg koda:

broj_ucenika > broj_ucenica

U ovom primeru podatke koji predstavljaju ukupan broj učenika u Srbiji, kao i broj učenica pridružili smo imenima ukupan_broj_ucenika i broj_ucenica. Imena koja smo dodali međurezultatima nazivaju se <b>promenljive</b>.

<h3>Minimum, maksimum, aritmetička sredina i medijana</h3>

<h3>Zadaci:</h3>

1. Kilogram jabuka košta 98 dinara, a cena banana je 134 dinara. 
Koliko je potrebno izdvojiti novca za 2 kilograma jabuka i 3 kilograma banana?


2. Za sklapanje auta - dečije igračke potrebno je 4 točka, 2 sedišta i 1 volan. Cena točka je 123 dinara, sedišta 67, a volana 89 dinara. Izračunaj koliko je potrebno novca da se sastavi 1000 komada ovih automobila.

<h3>Minimum, maksimum, aritmetička sredina i medijana</h3>

<p>Ocene jednog učenika možemo čuvati u promenljivama. </br></p>
<p>matematika = 5</p>
<p>programiranje = 5</p>
<p>fizika = 4</p>
<p>engleski_jezik = 5</p>
<p>I tako redom...međutim, mnogo je jednostavnije za različita izračunavanja ako imamo ocene objedinjene ja jednom mestu. Ocene učenika ćemo zapamtiti na sledeći način:</p>
<p>ocene = [5,5,4,5]</p>
<p>Ovakva struktura podataka naziva se <b>lista</b>.</p>


<h4>Minimum liste</h4>
<p>Neka je data lista nekih proizvoljnih brojeva:</p>
<p>lista = [1, 56, 3, 5, 12]</p>
Najveći element liste naziva se maksimum liste. Izračunavamo ga na sledeći način:
<p>najveci = max(lista)</p>
<h4>Maksimum liste</h4>
Najmanji element liste nativa se minimum liste. Izračunavamo ga na sledeći način:
<p>najmanji = min(lista)</p>
<h4>Suma</h4>
Zbir elemenata liste izračunavamo na sledeći način:
<p>zbir = sum(lista)</p>
<h4>Broj elemenata liste</h4>
Broj elemenata liste ( dužina liste) izračunavamo na sledeći način:
<p>duzina = len(lista)<p>

<h2>Zadaci</h2>

1. U listi temperatura = [12, 14, 10, 8, 16, 19, 10], date su temperature vazduha u jednoj nedelji. 
Odrediti najvišu i najnižu temperaturu u toj nedelji.

2. U listi prihodi = [120000, 13000, 210000, 190000,120000, 14000, 20000, 190000, 120000, 13000, 120000, 150000] date su ukupne mesečne zarade jedne porodice u toku jedne godine. Odrediti koliko je novca prihodovala ova porodica u ovoj godini.

3. U prethodnom periodu jedna kompanija je organizovala nekoliko konkursa na kojima je primljen određen broj zaposlenih. Broj zaposlenih
po konkursu dat je u sledećom listom zaposleni = [2, 14, 6, 20, 12, 8, 10, 3, 17]. Izračunati koliko je konkursa ova kompanija organizovala
kao i koliko je radnika zaposleno.

4. Ivana je položila sve ispite u prvom i drugom semestru. Ivanine ocene su date u sledećemom 
listom ocene = [10, 9, 10, 7, 9, 6, 8, 8, 6, 6, 10]. Izračunati Ivanin prosek na kraju 2. semestra.